### PR TITLE
WIP: #CU-c8hzvb change dependency from nbserverproxy to jupyter-server-proxy

### PR DESCRIPTION
**Pull request by andredias**

### CODE REVIEW

**Comment by hhuuggoo** [comment]
Task linked: [CU-c8hzvb switch from nbserverproxy to jupyter-server-proxy](https://app.clickup.com/t/c8hzvb)

File: templates/saturnbase-gpu/Dockerfile [hunk 1]
```diff
@@ -66,7 +66,7 @@ COPY install-jupyter.bash /tmp/install-jupyter.bash
 COPY jupyter.yml /tmp/jupyter.yml
 RUN bash /tmp/install-jupyter.bash
 
-RUN ${CONDA_DIR}/bin/jupyter serverextension enable --py nbserverproxy --sys-prefix
+RUN ${CONDA_DIR}/bin/jupyter serverextension enable jupyter-server-proxy --sys-prefix
```
Comment (jsignell) [inline templates/saturnbase-gpu/Dockerfile:69]: Based on the [readme](https://github.com/jupyterhub/jupyter-server-proxy) it seems like we should use the new method of installing like we do for `jupyterlab_bokeh` a few lines down.

Comment (hhuuggoo) [reply templates/saturnbase-gpu/Dockerfile:69]: > Based on the [readme](https://github.com/jupyterhub/jupyter-server-proxy) it seems like we should use the new method of installing like we do for `jupyterlab_bokeh` a few lines down.

I think that's only necessary/useful for the UI around jupyter server proxy apps?  To get the backend to do the proxying I think we still have to enable the serverextension?  labextensions only build JS iirc, and there is definitely backend python at play here.

**Review by hhuuggoo — APPROVED** [review]

File: templates/saturnbase-gpu/Dockerfile (hunk 1 above)
Comment (hhuuggoo) [reply templates/saturnbase-gpu/Dockerfile:69]: @jsignell my comment assumes you were talking about 

```
jupyter labextension install @jupyterlab/server-proxy
 
```

Comment (andredias) [reply templates/saturnbase-gpu/Dockerfile:69]: ok

Comment (jsignell) [reply templates/saturnbase-gpu/Dockerfile:69]: ok makes sense. Ignore me @andredias :facepalm:

**Review by jsignell — APPROVED** [review]

**Pull request merged**